SignalMessaging/ViewControllers/OWSTableViewController.m

```diff
@@ -457,9 +457,7 @@ - (void)loadView
     [self.tableView applyScrollViewInsetsFix];
 
     [self.view addSubview:self.tableView];
-    [self.tableView autoPinWidthToSuperview];
-    [self.tableView autoPinToTopLayoutGuideOfViewController:self withInset:0];
-    [self.tableView autoPinEdgeToSuperviewEdge:ALEdgeBottom];
+    [self.tableView autoPinEdgesToSuperviewEdges];
 
```

Comment:
This seems like something we'll want to test in the 10+ places we use this vc on iOS 9, 10 and 11 and on iPhone X.

Reply:
I'll add this in the test plan.